### PR TITLE
V22 rerun sample distribution figure

### PR DESCRIPTION
**Pull request by jharenza**
❗ stacked on #1401 

#### What scientific question is your analysis addressing?
Update code to rerun the sample distribution figure using `cancer_group_display`

#### What was your approach?
- Added an earlier merge of the palette into the histologies file for counting by cancer group display.
- Updated relevant parts of code to use cancer group display

#### What GitHub issue does your pull request address?
#1368 

#### Which areas should receive a particularly close look?
The assay legend is not printing, and I am not sure why.
Double check the code updates!

#### Is there anything that you want to discuss further?
See above.

#### What types of results are included (e.g., table, figure)?
sample distribution figures

#### What is your summary of the results?
The updates (minus assay panel) look to have been implemented.

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: figures/scripts/fig1-sample-distribution.R
Comment (sjspielman) [inline figures/scripts/fig1-sample-distribution.R:34]: Noting that we should add something into the 1B legend that unknown are excluded. I'll open that issue.

**Review by sjspielman — COMMENTED** [review]
👍 Looks good! Will wait until #1401 to formally approve but I'm not requesting any changes, unless we want to move ahead and totally remove the PDFs outputted to`supp/sample-distribution/` and associated code. These are not going to be used in the MS for sure. This is every line 212 and below in the figure script.

File: figures/scripts/fig1-sample-distribution.R
Comment (jharenza) [reply figures/scripts/fig1-sample-distribution.R:34]: this is a sneaky thing that comes and goes in new releases, since our clinical data is modified by sites all over the country at any given time 😅

**Comment by jharenza** [comment]
@sjspielman reran with the latest from #1401 - had to update a join, but looks good. also removed the code (and folder+plots) for the plots not in use.

**Comment by jharenza** [comment]
@jaclyn-taroni and @sjspielman this one is ready for re-review

**Comment by jharenza** [comment]
> We can remove this line from `fig1-sample-distribution.R` at this point -
> 
> ```
> library(ggpattern)
> ```
> 
> In fact, after `grep`ing the whole repo, it seems _this script_ was the only place `ggpattern` was needed. In theory we can remove this installation from the Dockerfile as well, which may also help in the fight against rate limits (#1469). I don't think I'd do that in this PR, however - if we want to do that, we should open separately.
> 
> Otherwise it looks good to me!

updated

**Review by sjspielman — APPROVED** [review]
👍

**Pull request merged**
